webui/module/Auth/view/auth/auth/login.phtml

```diff
@@ -58,7 +58,11 @@ $this->headTitle($title);
          echo '<div class="input-group">';
       }
       echo '<div class="input-group-addon"><span class="glyphicon glyphicon-log-in" aria-hidden="true"></span></div>';
-      echo $this->formSelect($form->get('director')->setAttribute('class','form-control selectpicker'));
+      if($multi_dird_env) {
```

Comment:
multi_dird_env: From which class is this variable?

Reply:
It is passed into the View from the AuthController, see line 217.